apis/go.mod

```diff
@@ -83,7 +83,7 @@ require (
 	github.com/prometheus/client_model v0.6.0 // indirect
 	github.com/prometheus/common v0.53.0 // indirect
 	github.com/prometheus/procfs v0.13.0 // indirect
-	github.com/rabbitmq/cluster-operator/v2 v2.11.0 // indirect
+	github.com/rabbitmq/cluster-operator/v2 v2.9.0 // indirect
```

Comment:
we have https://github.com/openstack-k8s-operators/renovate-config/blob/main/default.json5#L155 to keep it bellow 2.10. but the replace bellow will override it anyways